Source/WebKit/Platform/IPC/Connection.cpp

```diff
@@ -501,6 +501,8 @@ void Connection::invalidate()
 
 void Connection::markCurrentlyDispatchedMessageAsInvalid()
 {
+    RELEASE_LOG_FAULT(IPC, "Invalid message dispatched");
```

Comment:
Nit:  "Invalid IPC message dispatched".

The other option here would be to put in the `MESSAGE_CHECK_*` macros in `Source/WebKit/Platform/IPC/Connection.h` and do this to get unique signatures:

```
    RELEASE_LOG_FAULT(IPC, WTF_PRETTY_FUNCTION ": Invalid message dispatched");
```

I would actually prefer that because we'd get unique signatures for each issue.

Reply:
As the patch currently stands, all issues from all message checks would be aggregated under a single "Invalid message dispatched" signature.